libyul/YulString.h

```diff
@@ -117,10 +130,15 @@ class YulString
 	bool operator!=(YulString const& _other) const { return m_handle.id != _other.m_handle.id; }
 
 	bool empty() const { return m_handle.id == 0; }
-	std::string const& str() const
+	std::string const str() const
 	{
 		return YulStringRepository::instance().idToString(m_handle.id);
 	}
+	std::string const prefix() const
```

Comment:
If we don't add the suffix to the string in the repository we should at least return by reference here (see also the comment in the repository above).